erigon-lib/state/inverted_index.go

```diff
@@ -893,13 +893,19 @@ func (iit *InvertedIndexRoTx) DebugEFAllValuesAreInRange(ctx context.Context) er
 			}
 			if item.startTxNum > ef.Min() {
 				err := fmt.Errorf("DebugEFAllValuesAreInRange1: %d > %d, %s, %x", item.startTxNum, ef.Min(), g.FileName(), k)
```

Comment:
strange err description: "DebugEFAllValuesAreInRange1" and "DebugEFAllValuesAreInRange2" maybe better to explain it or drop magic numbers?